### DISCUSSION TITLE
OTP for revealing the key

### DISCUSSION THREAD

**Discussion by 0xab3d** [pglombardo/PasswordPusher#1942] [Ideas]
Hey, thanks for this great project. I have one suggestion. It would be awesome if we can add a recipient email to the pushed content so that an OTP will be sent before revealing the content and burning a view.

**Comment by pglombardo** [comment]
It's funny that you request this now.  I was just talking about adding this feature with another user yesterday.  :-)

I'll add this hopefully soon.

Would you want the OTP code automatically generated or do you want to specify it in "Passphrase Lockdown"?

**Reply by 0xab3d** [reply]
Thanks for the heads-up 😁 auto generated is better I think, just avoid any user intervention with the security part.
